iothub/service/src/RegistryManager.cs

```diff
@@ -620,10 +623,18 @@ public void Dispose()
         /// <param name="exportBlobContainerUri"></param>
         /// <param name="outputBlobName"></param>
         /// <param name="excludeKeys"></param>
-        /// <param name="cancellationToken"></param>
+        /// <param name="cancellationToken">Task cancellation token</param>
         /// <returns>JobProperties of the newly created job.</returns>
         public abstract Task<JobProperties> ExportDevicesAsync(string exportBlobContainerUri, string outputBlobName, bool excludeKeys, CancellationToken cancellationToken);
 
+        /// <summary>
+        /// Creates a new bulk job to export device registrations to the container specified by the provided URI.
+        /// </summary>
+        /// <param name="jobParameters">Parameters for the job</param>
+        /// <param name="cancellationToken">Task cancellation token</param>
+        /// <returns>JobProperties of the newly created job.</returns>
+        public abstract Task<JobProperties> ExportDevicesAsync(JobProperties jobParameters, CancellationToken cancellationToken = default);
```

Comment:
New API for customers for export job.

Reply:
Should there be tests for these APIs?

Reply:
Yes, but I can't run/test them until this new version is available in production.